esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -404,6 +406,7 @@ define([
                         context.thorProcess.set("value", targetData[0].ClusterGroup);
                 } else {
                    context.slaveLogs.set("disabled", true);
+                   context.includeSlaveLogsCheckbox.set("disabled", true);
```

Comment:
IOW you can download the slave logs independently of a ZAP report (which may be overkill?):  
![image](https://cloud.githubusercontent.com/assets/966863/14558862/f31beda0-02d2-11e6-9703-db1b9c48a3d2.png)